routes/web/project/vm.rb

```diff
@@ -34,10 +34,15 @@ class CloverWeb
 
     r.on "create" do
       r.get true do
+        require "yaml"
```

Comment:
Interesting, `require` at `lib/billing_rate.rb` is not enough.

Reply:
It is enough, this is artifact of my experimentation. Deleting it :)